lib/tldr-lint.js

```diff
@@ -27,7 +29,10 @@ module.exports.ERRORS = parser.ERRORS = {
   'TLDR103': 'Command example is missing its closing backtick',
   'TLDR104': 'Example descriptions should prefer infinitive tense (e.g. write) over present (e.g. writes) or gerund (e.g. writing)',
   'TLDR105': 'There should be only one command per example',
-  'TLDR106': 'Page title should start with a hash (\'#\')'
+  'TLDR106': 'Page title should start with a hash (\'#\')',
+  'TLDR107': 'File name should end with .md extension',
+  'TLDR108': 'File name should be all lowercase',
```

Comment:
Hrm, I'm still on the fence about enforcing lowercase filenames.

Reply:
Except that given that TLDR is aimed to run and be used on multiple platforms, and so has to aim for the platform of least capabilities, in this case Windows and macOS which are case insensitive, and where you cannot have both `common/R.md` and `common/r.md` pages and have them point to separate files.

Reply:
@MasterOdin looks like the discussion in tldr-pages/tldr#5085 was prematurely closed, if @sbrl still has doubts about the final solution. Let's discuss there to avoid splitting the threads.

Reply:
I've removed this change, this will evidently happen in a later PR.